Source/Core/Core/NetPlayServer.cpp

```diff
@@ -847,6 +911,122 @@ bool NetPlayServer::StartGame()
   return true;
 }
 
+void NetPlayServer::SetSyncSaveData(bool sync)
+{
+  m_sync_save_data = sync;
+}
+
+// called from ---GUI--- thread
+bool NetPlayServer::SyncSaveData()
+{
+  m_save_data_synced_players = 0;
+
+  {
+    // We always send this packet, as it tells the clients whether or not we're syncing saves
+    sf::Packet pac;
+    pac << static_cast<MessageId>(NP_MSG_SYNC_SAVE_DATA);
+    pac << static_cast<MessageId>(SYNC_SAVE_DATA_NOTIFY);
+
+    SendAsyncToClients(std::move(pac));
+  }
+
+  // TODO: Wii save handling
+  for (size_t i = 0; i < 2; i++)
+  {
+    if (m_settings.m_EXIDevice[i] == ExpansionInterface::EXIDEVICE_MEMORYCARD)
+    {
+      std::string path = i == 0 ? SConfig::GetInstance().m_strMemoryCardA :
```

Comment:
`i == 0` should likely be it's own `const bool`, given the comparison is also duplicated  at like 950